services/openshift/scripts/openshift_provision

```diff
@@ -120,5 +120,9 @@ if [ ! -f ${ORIGIN_DIR}/configured.user ]; then
   echo "[INFO] Adding required roles to openshift-dev and admin user ..."
   oadm policy add-role-to-user basic-user openshift-dev --config=${OPENSHIFT_DIR}/admin.kubeconfig
   oadm policy add-cluster-role-to-user cluster-admin admin --config=${OPENSHIFT_DIR}/admin.kubeconfig
+  oc login https://127.0.0.1:8443 -u openshift-dev -p devel \
+        --certificate-authority=${OPENSHIFT_DIR}/ca.crt &>/dev/null
+  oc new-project test --display-name="OpenShift 3 Sample" \
```

Comment:
maybe 'default' or 'openshift-dev-project' are better project names. And the description "Default project for openshift-dev". Just trying to be more concise. @mmalina wdyt? 

Otherwise I think it is a good idea to add this.